### PR TITLE
Clean up peer connection after each addTransceiver test

### LINKED ISSUE #8301: WebRTC tests should close RTCPeerConnection's at end of test
It looks like the current WPT tests for WebRTC do not call close() at the end of each test.

### PR DESCRIPTION
**Pull request by soareschen**
Fixes #8301.

Calls `t.add_cleanup()` in each test case to close the `RTCPeerConnection` to free up the resources.

We will test the effectiveness of cleaning up the peer connection after each test case. If the approach works well this can be applied to all other test files as well.

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/20818)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/304637122)

File: webrtc/RTCPeerConnection-addTransceiver.html
Comment (snuggs) [inline webrtc/RTCPeerConnection-addTransceiver.html]: Nice! ;-)

**Review by snuggs — APPROVED** [review]
Great way to clean up after ourselves @soareschen. Once this is merged we can figure out how to treat the other relevant tests. And I wouldn't recommend anything unless I was happy to assist as well. 😃

**Review by nils-ohlmeier — APPROVED** [review]
LGTM

**Comment by soareschen** [comment]
@foolip how can we add more contributor to the list of approved reviewers? There previous reviewers doesn't seem to be active.

**Review by snuggs — APPROVED** [review]
LGTM

**Comment by snuggs** [comment]
> @foolip how can we add more contributor to the list of approved reviewers? There previous reviewers for WebRTC don't seem to be active. - @soareschen 

Great question. Would like to know myself as have seen this on a few occasions with webrtc here.

**Comment by foolip** [comment]
@soareschen, I think it looks like there's a problem of too many reviewers for webrtc/, that means nobody in particular will feel responsible.

To add more people, just edit webrtc/OWNERS and make sure those people have write access to the repo.

**Review by agouaillard — APPROVED** [review]

**Comment by agouaillard** [comment]
nils approved, i'm merging.

**Comment by snuggs** [comment]
@foolip I am familiar with the process on a multitude of repos on github.  I have also followed, written code, and reviewed specs for webrtc as well. I am not saying I know all there is related to the subset of webrtc features in WPT. However, I am able to review on a (fairly) daily basis. I can figure out who is usually most active for review like @agouaillard here simply from extrapolating who was responsible for merges. Would love to have write access and can begin with some examples of my own. Am currently starting a project that is 100% webrtc and the fact WPT is moving forward with these tests was one of the reasons I'm less worried about testing than i was last year. Would love to help out wherever I can.

No harm no foul if declined. 🙏 

Enjoy the :turkey:

**Comment by agouaillard** [comment]
Soares is my employee, so I want to avoid conflict of interest by reviewing his patches. If others review, I will merge though.

**Comment by foolip** [comment]
@snuggs, that would make you one of the most qualified and enthusiastic reviewers anywhere in web-platform-tests, I'll do a PR to add you right away :) Much appreciated!

**Comment by foolip** [comment]
@snuggs, oh wait, you added yourself in https://github.com/w3c/web-platform-tests/pull/8143, you just need write access. Making it so!

**Comment by snuggs** [comment]
> @snuggs, oh wait, you added yourself in #8143, - @foolip 

On it like white on rice. 😎

**Pull request merged**
